private/bufpkg/bufplugin/bufplugindocker/docker_test.go

```diff
@@ -123,13 +124,16 @@ func buildDockerPlugin(t testing.TB, dockerfilePath string, pluginIdentity strin
 	}
 	imageName := fmt.Sprintf("%s:%s", pluginIdentity, stringid.GenerateRandomID())
 	cmd := command.NewRunner()
+	envContainer, err := app.NewEnvContainerForOS()
+	require.NoError(t, err)
```

Comment:
Not sure what changed, but it appears it needs to run with `PATH` at least. Just propagating the OS env vars through to running the command for the tests.